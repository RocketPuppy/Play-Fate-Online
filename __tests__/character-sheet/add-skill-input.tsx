import * as React from "react";
import * as renderer from "react-test-renderer";
import AddSkillInput from "../../src/character-sheet/add-skill-input";

const noop = () => {};

test("renders all the pieces", () => {
  const render = renderer.create(<AddSkillInput onSkillAdd={noop} />);

  expect(render.toJSON()).toMatchSnapshot();
});
