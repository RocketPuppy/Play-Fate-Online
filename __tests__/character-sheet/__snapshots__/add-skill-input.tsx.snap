// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<form>
  <input
    defaultValue=""
    onChange={[Function]}
    placeholder="Athletics"
    type="text"
  />
  <button
    disabled={true}
    onClick={[Function]}
  >
    Add Skill
  </button>
</form>
`;
